src/NServiceBus.Core/Scheduling/Schedule.cs

```diff
@@ -9,7 +9,7 @@ namespace NServiceBus
     /// <summary>
     /// Extends the context with scheduling capabilities.
     /// </summary>
-    public static class ScheduleBusExtensions
+    public static class ScheduleExtensions
```

Comment:
Do we need an obsolete here even though I can't come up with a reason why anyone would use the type in some way? (Knee jerk is no)